audbackend/core/backend.py

```diff
@@ -99,7 +100,9 @@ def exists(
             ``True`` if file exists
 
         Raises:
-            BackendError: if an error is raised on the backend
+            BackendError: if ``suppress_backend_errors`` is ``False``
+                and an error is raised on the backend,
+                e.g. ``path`` does not exist
```

Comment:
Good idea with the `e.g.`, now it's more obvious that the `if` statement belongs to everything that follows.